frontend/src/App.jsx

```diff
@@ -11,76 +11,164 @@ import jwtDecode from "jwt-decode";
 import "react-toastify/dist/ReactToastify.css";
 import "./App.css";
 
-class App extends Component {
-  state = {
-    isModalOpen: false,
-    isModalOpen2: false,
-  };
+// class App extends Component {
```

Comment:
please remove comment line the code should always be clean as it is going to be merged in master

Reply:
Yeah definitely, I just put them in the comment for the ease of the review. Anyway, I will remove them.